docs/widget-aem-bundle-info.md

```diff
@@ -0,0 +1,10 @@
+# [Home](/cogboard/) >> AEM Bundle Info widget
+
+#### Configuration:
+
+- `Endpoint` - choose AEM endpoint
+- `Schedule Period` - time interval between executions - recommended: `300` (5min)
+- `Error threshold for 'resolved' bundles` - how many bundles with status `resolved` are needed to throw an error instead of warning
+- `Error threshold for 'installed' bundles` - how many bundles with status `installed` are needed to throw an error instead of warning
+- `Excluded bundles` - list of bundles to be ignored when calculating widget status; can be either display name or symbolic name.
+  Excluded bundles will be put in a separate section, with option to view them all (including statuses).
```

Comment:
Please move this file content into `Wiki Home` as this is now our documentation place

Reply:
It's also there :)

Reply:
docs/* is no longer supported

Reply:
Removed.